Automattic/jetpack
# Stats: Update dependencies for stats admin bundle

#### Changes proposed in this Pull Request:
The PR changes to load the dependencies externalized by https://github.com/Automattic/wp-calypso/pull/71442

#### Other information:


#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Set up env with Option 2 in `PejTkB-3E-p2` <- use branch `update/optimize-bundle-size` 
* Do not forget to delete `dist` in package `stats-admin`
* Enable new Stats experience with instruction in `PejTkB-3E-p2`
* Open `/wp-admin/admin.php?page=stats`
* Ensure everything generally looks and feels okay
* Ensure links etc all work
* If anything's broken please raise bugs in [the project board](https://github.com/orgs/Automattic/projects/484)
